btrack/napari/constants.py

```diff
@@ -17,3 +17,5 @@
     "time_thresh",
     "apop_thresh",
 ]
+
+QWIDGETLISTITEM_TRISTATE = {True: 2, False: 0}
```

Comment:
you could use `QtCore.Qt.CheckState.Checked` rather than `QWIDGETLISTITEM_TRISTATE[True]` when setting the widget value, and when checking if the widget is checked you could just cast to `bool`